doc/pages/Packaging.md

```diff
@@ -98,6 +98,10 @@ build: [
   ["./configure" "--prefix=%{prefix}%"]
   [make]
 ]
+build-test: [
+  ["./configure" "--enable-tests"]
+  [make "test"]
+]
```

Comment:
It's worth documenting the `build-test:` field below, but I would rather keep the example as simple as possible, and more importantly close to the template provided by opam, so I'd rather keep this out.